core/services/keystore/keys/dkgsignkey/key.go

```diff
@@ -0,0 +1,92 @@
+package dkgsignkey
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/group/edwards25519"
```

Comment:
I see that other keys use `crypto/ed25519` library. Should we confirm with research if there is preferred library here?

Reply:
ah never mind. I see that we use kyber everywhere in the internal vrf repo

Reply:
We can use crypto/ed25519 however we need to have a method that returns the private key as a `kyber.Scalar` so I opted to use the kyber implementation instead.